tests/lax_numpy_test.py

```diff
@@ -2185,6 +2191,11 @@ def testAstype(self):
       for a_dtype in (default_dtypes + unsigned_dtypes + bool_dtypes)
       for dtype in (default_dtypes + unsigned_dtypes + bool_dtypes)))
   def testView(self, shape, a_dtype, dtype):
+    if jtu.device_under_test() == 'tpu':
+      if (a_dtype == onp.bool_ or dtype == onp.bool_
+          or _dtype_info(a_dtype).bits in [8, 16]
+          or _dtype_info(dtype).bits in [8, 16]):
```

Comment:
You could just use `dtype.itemsize` for this (measured in bytes), e.g.,
```
>>> np.dtype(np.int16).itemsize
2
```

Reply:
It doesn't work for ``bfloat16``

Reply:
```
In [7]: jnp.dtype(jnp.bfloat16).itemsize
Out[7]: 2
```

Seems to work for me?

Reply:
Ah, I guess I thought that ``jnp.bfloat16)`` was a dtype.